src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -675,6 +683,7 @@ private bool IsMapType()
             return DisplayHint == "map";
         }
 
+        [DebuggerHidden()]
```

Comment:
Is this just to keep the debugger from func-evaling the property? DebuggerHidden is a pretty big hammer. This code will no longer be debuggable, even with JMC off. I would suggest changing this to be a method instead of a property, as it looks like it really shouldn't be a property. If that isn't possible, I would suggest using Debugger.NotifyOfCrossThreadDependancy in the 'not-yet-fetched' case.